.ci/gcb-push-downstream.yml

```diff
@@ -181,6 +181,32 @@ steps:
       args:
           - $COMMIT_SHA
 
+    - name: 'gcr.io/graphite-docker-images/go-plus'
+      id: magician-check-vcr-cassettes
+      waitFor: ["tpgb-push"]
```

Comment:
Do we want to wait for cassette-merger instead?

Reply:
Done